apps/src/code-studio/pd/application_dashboard/quick_view_table.jsx

```diff
@@ -54,6 +54,9 @@ export default class QuickViewTable extends React.Component {
       },
       cell: {
         format: (status) => {
+          if (status === 'move_to_interview') {
+            return 'Move to Interview';
+          }
```

Comment:
not a big deal, but does this have to be a special case? Why not just 
```
  _.upperFirst(status.replace(/_/g, ' '))
```

?

And incidentally if we go with that, this can be condensed into an expression:
```jsx
format: (status) =>   _.upperFirst(status.replace(/_/g, ' ')),
```

Reply:
Because 'to' is supposed to be lower case :)